src/features/authentication/authenticationSlice.ts

```diff
@@ -0,0 +1,136 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import {
+  AuthStoreType,
+  AuthenticationSteps,
+} from "../../lib/authentication/authTypes";
+import {
+  Auth0AuthProvider,
+  authConfig,
+} from "../../lib/authentication/auth0AuthProvider";
+import {
+  MsalAuthProvider,
+  staticMsalConfig,
```

Comment:
this static doesn't look right.

(self remainder to check)

Reply:
it's because we don'ìt have Env yet